calico_containers/pycalico/datastore_datatypes.py

```diff
@@ -133,9 +134,16 @@ def __init__(self, cidr, ipip=False, masquerade=False, ipam=True):
         """
         # Normalize the CIDR (e.g. 1.2.3.4/16 -> 1.2.0.0/16)
         self.cidr = IPNetwork(cidr).cidr
+        self.ipam = bool(ipam)
+        if self.ipam:
```

Comment:
You should add a comment to the doc string to indicate the limitation on the CIDR when used with calico IPAM module